spec/fixtures/api/utility-process/no-js-after-exit.js

```diff
@@ -0,0 +1,2 @@
+process.exit(1);
+process.stdout.write('Unexpected');
```

Comment:
Is there any way to test it without the race condition of writing to stdout? See https://nodejs.org/docs/latest-v22.x/api/process.html#processexitcode

That said, it's unlikely we'll ever hit the race condition, so if there's no better way, that's okay.

Reply:
Updated in https://github.com/electron/electron/pull/47469/commits/95033edacd2a9ddb0a41d938081846ec94bed3ba

Reply:
Thank you! :)